libs/gi/localization/assets/locales/en/page_character_optimize.json

```diff
@@ -115,9 +134,39 @@
     "cancel": "Cancel",
     "generateBuilds": "Generate Builds"
   },
+  "buildShowingNum": "Showing <2>{{count}}</2> build generated for <6/>.",
+  "generatedOn": "Build generated on: ",
+  "selectChar": "Select a character to generate builds.",
+  "clearBuildsBtn": "Clear Builds",
+  "createBuildTc": {
```

Comment:
"createBuildTc", "createBuildReal" and "buildDisplay"  should go to `build.json`

Reply:
Completed. :)